pkg/scd/errors/errors.go

```diff
@@ -32,6 +32,16 @@ func MissingOVNsErrorResponse(missingOps []*dssmodels.Operation) (*spb.Status, e
 		}
 		detail.EntityConflicts = append(detail.EntityConflicts, entityRef)
 	}
+	for _, missingConstraint := range missingConstraints {
+		constraintRef, err := missingConstraint.ToProto()
+		if err != nil {
+			return nil, stacktrace.Propagate(err, "Error converting missing Constraint to proto")
+		}
+		entityRef := &scdpb.EntityReference{
+			ConstraintReference: constraintRef,
+		}
+		detail.EntityConflicts = append(detail.EntityConflicts, entityRef)
```

Comment:
nit: You could save a temporary here and do:
```
detail.EntityConflicts = append(detail.EntityConflicts, &scdpb.EntityReference{})
```

Reply:
Done.